lib/codegen/meta-python/cdsl/isa.py

```diff
@@ -316,7 +316,8 @@ class EncRecipe(object):
     :param name: Short mnemonic name for this recipe.
     :param format: All encoded instructions must have this
             :py:class:`InstructionFormat`.
-    :param size: Number of bytes in the binary encoded instruction.
+    :param base_size: Base number of bytes in the binary encoded instruction.
```

Comment:
I'm a little surprised to see this as a base plus extra bytes, rather than a max minus unneeded bytes, however the rest of this patch seems to demonstrate that we don't currently need the max size for anything. And, it is more intuitive to start with a base and add from there.

I'm trying to think of anything that we might want a max for. But before regalloc while, it might occasionally be useful to know how big things are, we won't have the fills and spills yet. And after regalloc, we can compute the actual sizes. So it may not matter for the foreseeable future.

Reply:
> And, it is more intuitive to start with a base and add from there.

This +1. Also, it's kind of an edge case when there's something to add; the most generic case is when we don't add anything.